### PR TITLE
[Caffe2] Revamp the convnet benchmark code by using models from model zoo

### PR DESCRIPTION
**Pull request by yinghai**
.

### CODE REVIEW

File: caffe2/python/ideep/test_ideep_net.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+from caffe2.proto import caffe2_pb2
+from caffe2.python import core, workspace
+from caffe2.python.models.download import downloadFromURLToFile
+import numpy as np
+import argparse
+import time
+import os.path
+import json
+
+class ModelFetcher:
```
Comment (houseroad) [inline caffe2/python/ideep/test_ideep_net.py]: Shall we lift this part out from two tests?

Comment (yinghai) [reply caffe2/python/ideep/test_ideep_net.py]: Where you want to put it? caffe2/python/model? :)

**Review by houseroad — APPROVED** [review]
Nit: refactor the download.py to merge _download and downloadModel together.

**Review by bddppq — APPROVED** [review]
LG

**Pull request merged**